### ISSUE TITLE
Remove `exclusive: false`

### ISSUE THREAD

**Issue by basil** [jenkins-infra/repository-permissions-updater#3883]
At the time of this writing, 67 out of 507 CD-enabled components (13%) have `exclusive: false`. There isn't a real use case for this, so most (if not all) of these represent technical debt of some kind—typically a failure to complete the process of CD enablement and release a version using CD; because if that had already taken place, then I would have enabled exclusive CD in #3877.

These should be systematically evaluated; if CD is enabled in the plugin repository but a CD release has not been performed, then a CD release should be performed (in which case `exclusive: false` should be deleted), and if CD was never enabled in the plugin repository, then the attempt to enable CD should be completed (culminating in a CD release) or CD disabled for that plugin in this repository.

**Comment by Brijeshthummar02** [comment]
as i see there are total of 65 yml files where it is exclusive: false , i'm removing all and set
 cd:
  enabled: false  # CD disabled since no release was performed

**Comment by MarkEWaite** [comment]
@Brijeshthummar02 I don't think that removing them all will address the statement of the issue.  Each of those 65 cases needs to be analyzed and handled as described:

> These should be systematically evaluated; if CD is enabled in the plugin repository but a CD release has not been performed, then a CD release should be performed (in which case exclusive: false should be deleted), and if CD was never enabled in the plugin repository, then the attempt to enable CD should be completed (culminating in a CD release) or CD disabled for that plugin in this repository.
